functions.php

```diff
@@ -166,3 +166,25 @@ function endsWith($haystack, $needle) {
 	return $needle === "" || (($temp = strlen($haystack) - strlen($needle)) >= 0 && strpos($haystack, $needle, $temp) !== FALSE);
 }
 
+function aggregate_newsletters($newslettersDir, $extension = "html"){
+	if ((file_exists($newslettersDir) && is_dir($newslettersDir)) === false)
```

Comment:
More like if (!file_exists or (file_exists and is_dir === false))
Otherwise it dismisses the case where the file/dir doesn't exists at all